Documentation/ApiOverview/UpdateWizards/Creation/Index.rst

```diff
@@ -0,0 +1,74 @@
+.. include:: ../../../Includes.txt
+
+.. _update-wizards-creation-generic:
+
+===============================
+Creating generic update wizards
+===============================
+
+Each update wizard consists of a single PHP file containing a single PHP class. This
+class has to extend :php:`TYPO3\CMS\Install\Updates\AbstractUpdate` and implement the
+abstract methods::
+
```

Comment:
"[...] abstract methods:"
* only one semicolon? Or is there a special feature going along with the 2 semicolons here?

Question: What do you mean by "implement the abstract methods:"? Isn't that already clear, since the AbstractUpdate is extended?

Reply:
Two semicolons are displayed as a single one, but start a code block. This is the short form of

```
.. _code-block:: php
```

Hopefully this is obvious to most developers yes. Should we omit this part of the sentence?

Reply:
"double semicolons": I understand, thank you very much  for the explanation!

Currently, the "implement the abstract methods" does not refer to anything and as a newcomer I wonder where these abstract methods that are mentioned next to he "extend AbstractUpdate" come from. A little moment of doubt, that can vanish, when the reference "its" is used. So if you indeed refer to the methods of AbstractUpdate, the following little change would make it a lot clearer to my mind: 
"This class has to extend :php:`TYPO3\CMS\Install\Updates\AbstractUpdate` and implement **_its_** abstract methods"
That way the reference to "AbstractUpdate" is clear. But only if that is technically correct!
A lot of explanation for just 3 letters ... :-) What do you think?

BTW: Removing it would be ok, too. At this point, the documentation addresses extension developer, who already update their existing extension. So it is fair to assume that they already know what it means to extend an abstract class.